source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -197,8 +295,5 @@ namespace AdaptiveSharedNamespace
         ActionsConfig actions;
         ContainerStylesDefinition containerStyles;
         MediaConfig media;
```

Comment:
each of these private members should be prepended with a `_` to make it clear that they're private.